src/util/mapbox.js

```diff
@@ -119,3 +125,72 @@ function formatUrl(obj: UrlObject): string {
     const params = obj.params.length ? `?${obj.params.join('&')}` : '';
     return `${obj.protocol}://${obj.authority}${obj.path}${params}`;
 }
+
+const STORAGE_TOKEN = 'mapbox.userTurnstileData';
+const localStorageAvailable = storageAvailable('localStorage');
```

Comment:
There is no fallback for saving the last update time locally - it would send multiple requests. Are you thinking that there should be a local variable for `lastUpdateTime` so that in a given instantiation of `Map`, this still maintains the _once per day_ frequency?